docs/en/user/repo/branches.md

```diff
@@ -22,7 +22,9 @@ New branches can be created with the "Create Branch" form. There, you have to ch
 ![Create Branch](assets/repository-create-branch.png)
 
 ### Branch Details Page
-This page shows some commands to work with the branch on the command line.
+This page shows some commands to work with the branch on the command line. The ahead/behind commits can be found directly
+beneath the name of the branch.
```

Comment:
This is a little bit ambiguous. I would prefer something like "The number of commits ahead/behind can..."